common/views/components/IIIFPresentationPreview/IIIFPresentationPreview.js

```diff
@@ -256,17 +251,14 @@ const IIIFPresentationDisplay = ({
   iiifPresentationLocation,
   itemUrl,
 }: Props) => {
-  const [loaded, setLoaded] = useState(false);
+  const [viewType, setViewType] = useState('unknown');
```

Comment:
See my previous question - but if we keep it a string, assuming we want them to be the same?

Reply:
I take it back, I can see why we've got the three states.
Just to clarify, on the item page
* `unknown` === `can't look it up`
* `iiif` | `pdf` ===  `looked up manifest, can render`
* `none` === `looked it up and know we can't render it`

Reply:
exactly that

Reply:
Could we type it as a union type and leave a note as I'm not 100% sure it would be immediately obvious to our future selves ?